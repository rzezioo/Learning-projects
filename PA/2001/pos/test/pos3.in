20 46
32
32
25
15
64
49
87
28
72
85
75
79
56
85
97
37
86
31
81
80
5 7
9 10
15 17
1 10
10 9
20 19
6 5
14 16
19 20
2 1
13 15
6 8
11 13
16 15
12 11
13 14
4 3
20 10
10 12
7 8
15 16
18 20
2 4
16 18
1 2
4 6
8 7
9 11
7 9
17 18
11 12
18 17
2 1
5 6
1 1
14 13
3 5
8 10
3 4
1 20
20 19
20 20
17 19
1 3
10 10
12 14
