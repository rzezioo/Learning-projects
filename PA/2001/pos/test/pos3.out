46
1 2 1 1 10 9 10 12 11 13 15 16 14 13 14 12 11 9 7 5 6 8 7 8 10 10 20 19 20 18 16 15 17 18 17 19 20 20 1 2 4 3 5 6 4 3 1